doc/api/util.md

```diff
@@ -919,6 +988,34 @@ encoded bytes.
 
 The encoding supported by the `TextEncoder` instance. Always set to `'utf-8'`.
 
+## util.triggerNodeReport([filename])
+<!-- YAML
+added: v11.0.0
+-->
+
+* `filename` {string} The file to write into. **Default:** an empty string.
+* Returns: {string} 
+
+Returns the filename of the generated report.
```

Comment:
These are always sync, correct? If so, that should be noted.

Reply:
addressed, the documentation reflects its`synchronous` now.